src/hevm/src/EVM.hs

```diff
@@ -1905,12 +1908,63 @@ cheatActions =
               accessStorage a' (mksym slot) $ \res -> do
                 assign (state . returndata . word256At 0) res
                 assign (state . memory . word256At outOffset) res
+          _ -> vmError (BadCheatCode sig),
+
+      action "sign(uint256,bytes32)" $
+        \sig outOffset _ input -> case decodeStaticArgs input of
+          [sk, hash] ->
+            forceConcrete (mksym sk) $ \sk' ->
```

Comment:
I don't really get what you are doing here, tbh. `forceConcrete (mksym sk) $ \sk' ->` seems like a very arcane noop

Reply:
nvm, I get it. Root problem is `decodeStaticArgs` destroys the whiff information